openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -105,7 +131,7 @@ def with_inline_css(html_without_css):
     """
     css_filepath = settings.NOTIFICATION_EMAIL_CSS
     if not css_filepath.startswith('/'):
-        css_filepath = finders.FileSystemFinder().find(settings.NOTIFICATION_EMAIL_CSS)
+        css_filepath = file_path_finder(settings.NOTIFICATION_EMAIL_CSS)
```

Comment:
There is no need to define new single line function you can use built in function directly.